apps/examples/testcase/le_tc/kernel/tc_task.c

```diff
@@ -414,6 +417,108 @@ static void tc_task_prctl(void)
 	TC_ASSERT_EQ_ERROR_CLEANUP("prctl", strncmp(getname, setname, CONFIG_TASK_NAME_SIZE), 0, get_errno(), prctl(PR_SET_NAME, oldname, 0, 0, 0));
 
 	prctl(PR_SET_NAME, oldname, 0, 0, 0);
+
+#ifdef CONFIG_MESSAGING_IPC
```

Comment:
Those(messaging, preference, task monitor) are not kernel functionality, they are frameworks.
So there are their own samples. We can use them to check the coverage, we don't need to test from kernel testcases.

Reply:
Could you check @jeongchanKim 's comment ?